xinference/api/restful_api.py

```diff
@@ -1014,6 +1015,7 @@ async def launch_model(
             "replica",
             "n_gpu",
             "request_limits",
+            "multimodal_projector",
```

Comment:
This param is too specific, can we just put it into the kwargs? and let the llama.cpp engine handle it.